src/parsing/parsing.js

```diff
@@ -128,7 +130,10 @@
       }
     }
     var parsed = parser(string.substr(0, n));
-    return parsed == undefined ? undefined : [parsed, string.substr(n)];
```

Comment:
Oh. Well either way should be fine then. I wouldn't bother going and making them consistent.